lib/maze/option/parser.rb

```diff
@@ -41,7 +41,11 @@ def parse(args)
                 type: :boolean,
                 default: true
 
-            opt Option::REPEATER_API_KEY,
+            opt Option::ASPECTO_REPEATER_API_KEY,
+                'Enables forwarding of all received POST requests to Aspecto, using the API key provided.  MAZE_ASPECTO_REPEATER_API_KEY may also be set.',
+                type: :string
+
+            opt Option::BUGSNAG_REPEATER_API_KEY,
                 'Enables forwarding of all received POST requests to Bugsnag, using the API key provided.  MAZE_REPEATER_API_KEY may also be set.',
```

Comment:
MAZE_BUGSNAG_REPEATER_API_KEY?

Reply:
I considered it, but it would have been a breaking change.  I also considered having that as an alternative, but on balance found it unnecessary - the Aspecto forwarder is more niche and the existing one is implicitly "BugSnag".